apps/fluent-tester/src/FluentTester/TestComponents/Avatar/BasicAvatar.tsx

```diff
@@ -20,14 +20,17 @@ const avatarActiveAppearance: AvatarActiveAppearance[] = ['ring', 'shadow', 'glo
 
 const allSizes: WithUndefined<AvatarSize>[] = [
   undefinedText,
-  'size8',
+  'size20',
```

Comment:
I wonder if we can import AvatarSize here and use it to populate the picker?

Also would this break the test on Mac?

Reply:
Same comment about using AvatarPresence to populate the presence entries below

Reply:
I was trying to do that with previous Avatar task, there were errors. But maybe there's a way :) I'll try again and will let you know

Reply:
Oh okay sorry, I forgot that you had tried. No worries then.